apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx

```diff
@@ -1,12 +1,22 @@
+import { useState } from "react";
 import { Styled } from "./Navbar.styled";
 import MenuIcon from "@mui/icons-material/Menu";
 import { GenericButton } from "../../../../../../../../packages/ui/components/elements/Buttons/Button";
+import Modal from "../../../../../../../../packages/ui/components/elements/Modal/GenericModal";
 import router from "next/router";
 
 const NewNav = () => {
+  const [modalOpen, setModalOpen] = useState(false);
+  const closeModal = (): void => {
+    setModalOpen(false);
+  };
+
+  const openModal = (): void => {
+    setModalOpen(true);
+  };
   return (
     <Styled.Navbar>
-      <div className="nav-end">
+      <div className="nav-end" onClick={() => setModalOpen(true)}>
```

Comment:
So you got 2 `onclick` one in the parent and another in the child? but why? and you are setting both to TRUE?